### ISSUE TITLE
doesn't work for ColorTypes already shown

### ISSUE THREAD

**Issue by bjarthur** [JuliaImages/ImageInTerminal.jl#7]
should at least document, if not fix somehow, the fact that ImageInTerminal only works for ColorTypes that have not already been shown:

```
julia> using ColorTypes

julia> rand(Gray{Float64},3,3)
3×3 Array{ColorTypes.Gray{Float64},2}:
 Gray{Float64}(0.0766381)  Gray{Float64}(0.622001)   Gray{Float64}(0.155463)
 Gray{Float64}(0.422821)   Gray{Float64}(0.65208)    Gray{Float64}(0.261956)
 Gray{Float64}(0.0723423)  Gray{Float64}(0.0386245)  Gray{Float64}(0.627463)

julia> using ImageInTerminal

julia> rand(Gray{Float64},3,3)
3×3 Array{ColorTypes.Gray{Float64},2}:
 Gray{Float64}(0.697702)  Gray{Float64}(0.82697)   Gray{Float64}(0.942118)
 Gray{Float64}(0.844839)  Gray{Float64}(0.916802)  Gray{Float64}(0.415631)
 Gray{Float64}(0.940266)  Gray{Float64}(0.768606)  Gray{Float64}(0.845324)

julia> rand(Gray{Float32},3,3)
3×3 Array{Gray{Float32},2}:
██████
██████
██████
```

**Comment by Evizero** [comment]
I think that is a general Julia theme and not specific to this package. At least as far as I understand v0.5. Once code is JIT compiled it is compiled and importing a new package won't change that.

I think 0.6 brings some changes to that, but I have not looked into that at all.

**Comment by timholy** [comment]
Yes, I can confirm this is fixed on julia 0.6.
